src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -342,7 +342,7 @@ class Microsoft::Terminal::Core::Terminal final :
     std::function<void(bool)> _pfnShowWindowChanged;
     std::function<void(const int, const int, const std::chrono::microseconds)> _pfnPlayMidiNote;
     std::function<void(std::wstring_view, unsigned int)> _pfnCompletionsChanged;
-    std::function<void(std::wstring_view)> _pfnSearchMissingCommand;
+    std::function<void(std::wstring_view, const til::CoordType&)> _pfnSearchMissingCommand;

```

Comment:
Only large classes (like a large array) or those expensive to copy (like `wstring` - creates a complete, separate copy of the entire string no matter how large) should be passed by reference. Since `CoordType` is a primitive integer, we should pass it by value. This results in it being passed in a CPU register and also makes debugging simpler.

Only a few types are in the gray area in between (namely reference counted types: `com_ptr`, WinRT instances, `hstring`, etc. They're slightly expensive to copy but not terribly so.)